src/components/link/README.md

```diff
@@ -22,11 +22,19 @@ To generate a `<router-link>` instead, specify the route location via the `to` p
 Router links support various additional props.  Refer to the [Router support](/docs/reference/router-links)
 reference section for details.
 
+If your app is running under [Nuxt](https://nuxtjs.org), the [`<nuxt-link>`](https://nuxtjs.org/api/components-nuxt-link)
+component will be used instead of `<router-link>`.  The `<nuxt-link>` component supports all the
```

Comment:
`<router-link>`.  The `<nuxt-link>` => `<router-link>`. The `<nuxt-link>`

Double space after the `.`

Reply:
Fixed